schemas/public/onpremises-kfd-v1alpha2.json

```diff
@@ -635,6 +639,10 @@
         "relativeVendorPath": {
           "type": "string",
           "description": "The relative path to the vendor directory, does not need to be changed"
+        },
+        "customRegistry": {
```

Comment:
what if we call this `registry` instead of `customRegistry`? I think is cleaner.

Also, if we call it `registry`, we can then (out of scope for this PR) set a default value for it like `registry.sighup.io` and use that variable instead of "hardcoding" reigstry.sighup.io in the templates and modules; and then if the user sets a different value it would be picked up automatically instead of using the current approach with `sed`.